### DISCUSSION TITLE
Export findings in asff format

### DISCUSSION THREAD

**Discussion by anandg112** [aquasecurity/starboard#634] [Development]
As a user of Starboard, I would like findings from scans such kube-hunter, CIS kubernetes benchmark and configuration audits to be exported in ASFF format (https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-findings-format.html) so the findings can be shipped to AWS SecurityHub. This will allow our security team to view findings easily instead of using low-level tooling like `kubectl` or `k9s`. Open to contributing, would need some guidance though.

**Comment by danielpacak** [comment]
👋 @anandg112 This sound to me like a great idea. Could you elaborate on how the end to end workflow of submitting ASFF files would look like:
- [ ] Would you use Starboard CLI and / or Starboard Operator?
- [ ] Would you export single CRD instances or you want to group them somehow (e.g. per namespace)?

Starboard CLI does support export to HTML so it seems that we could add a new option to specify output format of ASFF and (default) HTML. We'd be more than happy to guide you through the code once we agree on the scope of this functionality.
